src/Evolution/Systems/Cce/InterfaceManagers/GhInterpolationStrategies.hpp

```diff
@@ -9,8 +9,26 @@
 
 
 namespace Cce {
+/// Contains utilities for collecting, interpolating, and providing worldtube
+/// data for CCE that originates from other components in a running simulation.
 namespace InterfaceManagers{
 
+/*!
+ * \brief Enumeration of possibilities for the collection of worldtube data that
+ * will be collected by the interpolator.
+ *
+ * \details The available strategies are:
+ * - `InterpolationStrategy::EveryStep` : interpolation data will be provided on
+ * each full step for the time-stepper, so the worldtube data represents
```

Comment:
could you please indent each line of the items by two more space (so it is aligned with the `)